### ISSUE TITLE
Domain does not point to this server.

### ISSUE THREAD

**Issue by lordfiSh** [nextcloud/all-in-one#20]
Hi, 
not sure what is wrong here, Nextcloud Docker (nextcloud-aio-domaincheck) is reachable via Port 443 and showing a ID or something 
Yet the AIO Tools says "Domain does not point to this server." Any Hits what the Problem? 

![image](https://user-images.githubusercontent.com/389680/144255393-76fb2cf4-2bd7-4d10-8ea3-fd638017b29a.png)

**Comment by szaimen** [comment]
Hi, thanks for your request! Are you using a reverse proxy as ssl proxy or something like that? it looks like you have a valid certificate already at this step for your domain although it shouldn't have it at this point.

**Comment by lordfiSh** [comment]
Yes, i'm using Nginx Proxy Manager with a valid certificate (see https://github.com/nextcloud/all-in-one/issues/15)  

I thought I need to go trough the "wizzard" since no other containers are up yet, 
```
root@ubuntu01:/home/test# sudo docker ps | grep next
d689a7edee36   nextcloud/aio-domaincheck:latest               "/start.sh"              5 hours ago    Up 5 hours             0.0.0.0:443->443/tcp, :::443->443/tcp            nextcloud-aio-domaincheck
6354d4271baa   nextcloud/all-in-one:latest                    "start.sh /usr/bin/s…"   5 hours ago    Up 5 hours             0.0.0.0:80->80/tcp, :::80->80/tcp, 0.0.0.0:8080->8080/tcp, :::8080->8080/tcp, 0.0.0.0:8443->8443/tcp, :::8443->8443/tcp    nextcloud-aio-mastercontainer
root@ubuntu01:/home/test# curl -k http://0.0.0.0:80
curl: (56) Recv failure: Connection reset by peer
```

**Comment by szaimen** [comment]
Does nextcloud work after you've skipped this step?

**Comment by lordfiSh** [comment]
You cant skip it, that's the Problem.
I needed to get AIO Wizzard and get my Cert, after that every worked fine.

**Comment by szaimen** [comment]
See https://github.com/nextcloud/all-in-one/issues/15#issuecomment-983971267

**Comment by szaimen** [comment]
I changed my mind. Sry for the harsh answer!

Does it work if you change the `http://` in this line to `https://` ?

https://github.com/nextcloud/all-in-one/blob/13360b176e58deca026e8a190ab221131ac96549/php/src/Data/ConfigurationManager.php#L126

The file should be located inside the mastercontainer here: `/var/www/docker-aio/php/src/Data/ConfigurationManager.php`

I wonder especially if Nextcloud and all plugins like Office, Talk & Co. work afterwards and if you are able to log in with the android and desktop clients on your new Nextcloud instance?

If yes, I would accept a PR that allows `https://`, too.

**Comment by arsaboo** [comment]
I am seeing the same error with the latest image with Nginx proxy manager. Any resolution?

**Comment by szaimen** [comment]
Did you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md ?

**Comment by szaimen** [comment]
Based on that there is this: https://github.com/nextcloud/all-in-one/issues/317#issuecomment-1060623691

**Comment by arsaboo** [comment]
So, that helped. Here are configuration screens for anyone struggling:

![image](https://user-images.githubusercontent.com/18319734/158035127-3651506d-e318-4634-9832-76a005ad21fb.png)

![image](https://user-images.githubusercontent.com/18319734/158035312-822bcaac-00d7-42dc-ab96-f24aaa727591.png)

Just a quick follow up question - does the AIO not include elastic search?

**Comment by szaimen** [comment]
> does the AIO not include elastic search?

Not yet. See https://github.com/nextcloud/all-in-one/issues/78
